templates/_helpers.tpl

```diff
@@ -100,14 +100,21 @@ Shared environment block used across each component.
   value: "{{ .Values.postgresql.postgresqlDatabase }}"
 {{- end }}
 {{- if not .Values.redis.enabled }}
-- name: REDASH_REDIS_URL
-  {{- if .Values.externalRedisSecret }}
+- name: REDASH_REDIS_PASSWORD
+  {{- if .Values.externalRedis.auth.enabled }}
   valueFrom:
     secretKeyRef:
-      {{- .Values.externalRedisSecret | toYaml | nindent 6 }}
+      name: {{ .Values.externalRedis.auth.secretName | quote }}
+      key: {{ .Values.externalRedis.auth.secretKey }}
   {{- else }}
-  value: {{ default "" .Values.externalRedis | quote }}
+  value: ""
   {{- end }}
+- name: REDASH_REDIS_HOSTNAME
+  value: {{ .Values.externalRedis.host }}
+- name: REDASH_REDIS_PORT
+  value: "{{ .Values.externalRedis.port }}"
+- name: REDASH_REDIS_DB
```

Comment:
Why add this var if never set? cf https://github.com/blablacar/redash-contrib-helm-chart/pull/7/files#diff-8377b3e3740a3fcd9f682e5fb55425f2fdbece1791854b9e5013e7f1a5e60e7eR547